de-lib/src/main/java/org/iplantc/de/pipelineBuilder/client/json/App.java

```diff
@@ -24,6 +24,7 @@
     private List<PipelineAppData> outputAppData;
 
 	private int publicId;
+	private String systemId;
 	private String ID;
     private String templateId;
```

Comment:
Minor thing that's probably been there for a while, but looks like the spacing is a little wacky in this file?